### ISSUE TITLE
Brightness and keyboard options not working

### ISSUE THREAD

**Issue by CNSabata** [malensek/3RVX#64]
So yeah, as the title says, the brightness and keyboard options are not working for me. Whenever I check Caps Lock, Num Lock, Scroll Lock and Media Keys, I click apply and ok, and when I open back the settings, they're unchecked. So basically, I have 30% of the software working... Kind of depressing considering it's been my favorite osd on xp.
Is this related to me running Windows 10?

**Comment by SGvagon** [comment]
Confirm this problem. Brightness and keyboard notifications not working :( . And when I try set run program %windir%\system32\mblctr.exe to open Windows mobility center (Win + X default in Windows shortcut) , it doesn't work too.

**Comment by Laserbean** [comment]
I created an account for this. Please fix it Mr. Malensek. Same problem here. Also using Windows 7 Pro 64 bit.

**Comment by Valinwolf** [comment]
I can confirm that this is still an issue on v2.9.2 under Win10 Pro 64bit.

**Comment by Valinwolf** [comment]
Instead of creating a new issue like I said in #99, I will use this one since it is technically what my question is about. I was poking around in the code and I managed to get these settings to save and load, however they were not saving or loading because they were not implemented. I actually could not seem to find anything more than a log function (KeyboardOSD.cpp [line 62](https://github.com/malensek/3RVX/blob/f6ba63d52884e8aabce7a71fcfed27212f362a12/3RVX/OSD/KeyboardOSD.cpp#L62) and [line 67](https://github.com/malensek/3RVX/blob/f6ba63d52884e8aabce7a71fcfed27212f362a12/3RVX/OSD/KeyboardOSD.cpp#L67)) for the keyboard key options. Did I miss something?

TL;DR: Is the keyboard OSD features even implemented yet?

**Comment by JunedKhan101** [comment]
Facing the same issue it doesn't seem to work at all On what OS or environment it was previously running properly? I'm using Windows 11 Home.
